### ISSUE TITLE
Feature Request: Font size device + state

### ISSUE THREAD

**Issue by gasinkas** [sbormann/ioBroker.iqontrol#222]
**Question**
Hi! Is it possible to change the font size of the texts in the device tile?

I found in options a way to change the font size in toolbar and headers, but not for device texts (device name + device state + INFO_A + INFO_B ). Is it not possible or am I blind? :D

**Versions:**
 - Adapter version: 1.11.0 (latest)
 - JS-Controller version: 3.3.15
 - Node version: v14.18.irgendwas
 - Operating system: Linux

LG Stefan

**Comment by sbormann** [comment]
Hi, no it is noch possible at the moment, as the space available for these texts is fixed (changing size would mean you need to change the tile size and the positioning on the tile itself also)

**Comment by gasinkas** [comment]
Thanks for your answer! Especially on smaller devices with HD display (like the piPad) it's hard to read the device state. 

When I set the tile size to 1x2 there should be enough space for a bigger font size? If the text gets larger than possible to display, it could marquee like longer tile states already do. Just an idea....

**Comment by sbormann** [comment]
Absolutely, here is much room for improvement. My idea is to make the positions of elements inside the tile configurable - but that is a huge thing and needs much time. But it is on my list!

**Comment by gasinkas** [comment]
Allright, I love your idea :) Thanks for your work and I'm happy to see the project growing 😊 Frohes Neues!
